src/chromo/common.py

```diff
@@ -418,6 +418,12 @@ def to_hepmc3(self, genevent=None):
         """
         import pyhepmc  # delay import
 
+        if not pyhepmc.__version__ >= "2.13.2":
```

Comment:
Is this right way to compare versions?
There is a typo in pyhepm.

Reply:
Changed it to a more safe way with `packaging.version`